### PR TITLE
View playlist psp

### PR DESCRIPTION
**Pull request by PaulPatterson48**
## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

### CODE REVIEW

File: pages/playlist/[id].js [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+import PlaylistForm from '../../components/forms/PlaylistForm';
+
+export default function ViewPlaylists() {
+  const [editItem, setEditItem] = useState({});
+  const router = useRouter();
+
+  const { id } = router.query;
+
+  useEffect(() => {
+    viewPlaylistDetails(id).then(setEditItem);
```
Comment (taylorseager) [inline pages/playlist/[id].js]: When I run this on my local machine, I get `TypeError: (0 , _api_mergedData__WEBPACK_IMPORTED_MODULE_3__.default) is not a function`

File: pages/playlist/[id].js [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+  }, [id]);
+
+  return (
+    <PlaylistForm playlistObj={editItem} />
```
Comment (taylorseager) [inline pages/playlist/[id].js]: We should be returning the single playlist card with the associated podcast cards, not the playlist form. :)

**Review by taylorseager — CHANGES_REQUESTED** [review]
I left comments on the dynamic file for the view playlist details. The useEffect is throwing a runtime error and I think we should be returning the playlist card with the associated podcast cards :)

**Comment by PaulPatterson48** [comment]
Added viewPlaylistDetails

File: pages/playlist/[id].js (hunk 1 above)
Comment (PaulPatterson48) [reply pages/playlist/[id].js]: I think we solved this issue last night

File: pages/playlist/[id].js (hunk 2 above)
Comment (PaulPatterson48) [reply pages/playlist/[id].js]: We fixed this last night

**Pull request merged**